openslides_backend/presenter/presenter.py

```diff
@@ -120,16 +117,13 @@ def parse_presentations(self, payload: Payload) -> PresenterResponse:
         )
         response = []
         for presentation_blob in payload:
-            # user_id = presentation_blob["user_id"]
-            if presentation_blob["presentation"] == "dummy":
-                response.append(self.dummy_presentation())
+            Presenter = presenters.get(presentation_blob["presenter"])
+            if Presenter is not None:
+                p = Presenter()
```

Comment:
Here the same. I would prefere `presenter` or `presenter_instance` or something like that instead of `p`. But this is personal style of course I won't insist on that.

Reply:
Usually I decide on scope: the wider the scope of a variable the more descriptive its name. But I adapt in order to provide a consistent style.